grapher/dataTable/DataTable.tsx

```diff
@@ -451,7 +452,7 @@ export class DataTable extends React.Component<{
     }
 
     @computed private get entityNames() {
-        let tableForEntities = this.table.rootTable
+        let tableForEntities = this.table
```

Comment:
I wasn't exactly sure about the behavior here. I forget exactly but I think some countries were getting filtered out at the Grapher level but we wanted to show them in the DataTable. Maybe that's outdated, but I vaguely remember adding the `rootTable` to match prod.

Reply:
Ah, yes. This might be related to having the ability to filter small countries in the Table. Unfortunately, with the new tab design this control can't even be seen on the Table now (it used to be inline with the tabs).

We'll have to come back to that, but for now the rootTable didn't have the columns from the CovidExplorer.